pkg/etc/init.d/wavefront-proxy

```diff
@@ -63,6 +63,33 @@ if [[ -r $proxy_launch_conf ]]; then
 	fi
 fi
 
+if [[ ! -r $JAVA_HOME/bin/java ]]; then
```

Comment:
JAVA_HOME is actually overridden in the script to /opt/wavefront/wavefront-proxy/proxy-jre [(link)](https://github.com/basilisk487/java/blob/5600e881ee1c999e54c0a88c15745b3db4f52964/pkg/etc/init.d/wavefront-proxy#L29), so I believe these scenarios are not applicable here. 

However, it is indeed a good idea to allow user-defined JREs as well - added as $PROXY_JAVA_HOME.